ext/har/types.go

```diff
@@ -0,0 +1,356 @@
+// Original implementation from abourget/goproxy, adapted for use as an extension.
+// HAR specification: http://www.softwareishard.com/blog/har-12-spec/
+package har
+
+import (
+    "bytes"
+    "io"
+    "net/http"
+    "net/url"
+    "mime"
+    "net"
+    "strings"
+    "time"
+    "github.com/elazarl/goproxy"
```

Comment:
```suggestion

    "github.com/elazarl/goproxy"
```

Separate stdlib from all the other libraries